pkg/server/test/write.go

```diff
@@ -832,6 +860,54 @@ var writeCommandTests = []writeCommandTest{
 			},
 		),
 	},
+	{
+		_name: "ObsoleteModel1.0",
+		// state
+		model: &openfgapb.AuthorizationModel{
+			Id:            ulid.Make().String(),
+			SchemaVersion: typesystem.SchemaVersion1_0,
+			TypeDefinitions: []*openfgapb.TypeDefinition{
+				{
+					Type: "repo",
+					Relations: map[string]*openfgapb.Userset{
+						"admin":  {Userset: &openfgapb.Userset_This{}},
```

Comment:
`typesystem.This()` for short

This comment applies here and elsewhere where you use `&openfgapb.Userset_This{}`